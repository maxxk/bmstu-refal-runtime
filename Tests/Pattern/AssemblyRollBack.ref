$ENTRY Go
{
	= <Prout <RollBack a a a a a>>
	  <Prout <NoRollBack a a a a a>>	  	  
}

RollBack
{
	e.1 a e.2, e.1 : a = Success;
}

NoRollBack
{
	e.1 a e.2 = e.1 : a = Success;
}

