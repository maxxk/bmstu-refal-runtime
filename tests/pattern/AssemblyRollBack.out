Success 
[VM]: Func call failed!
Func call term:
<NoRollBack a a a a a >
Field of view:
<Prout <NoRollBack a a a a a >>
